help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
+| text                                                                                                                           	| first author only if<br>number of authors > 2<br>otherwise both authors 	| **%H**     |                      	| -Henneken<br>-Kurtz and Accomazzi                                                                                                  	|  -Henneken Muench<br>-Kurtz and Accomazzi                                                    	|
+| symbol                                                                                                                         	| first author only if<br>number of authors > 2<br>otherwise both authors 	| **%h**     |                      	|  -Henneken<br>-Kurtz & Accomazzi                                                                                                    	|  -Henneken Muench<br>-Kurtz & Accomazzi                                                             	|
+| text                                                                                                                           	| lastname                                                         	| **%M**     | , et al.             	|  -Henneken, Muench, Holm Nielsen, Blanco-Cuaresma, and Accomazzi<br>-Kurtz and Accomazzi                                            	|  -Henneken, Muench et al.-<br>Kurtz and Accomazzi                                            	|
+| symbol                                                                                                                         	| lastname                                                         	| **%m**     | , et al.             	|  -Henneken, Muench, Holm Nielsen, Blanco-Cuaresma, & Accomazzi<br>-Kurtz & Accomazzi                                                	|  -Henneken, Muench et al.<br>-Kurtz & Accomazzi                                              	|
+| none                                                                                                                           	| first author only lastname+                                      	| **%n**     |                      	|  -Henneken,+<br>-Kurtz,+                                                                                                            	|  -Henneken,+<br>-Kurtz,+                                                                     	|
+| text                                                                                                                           	| lastname                                                         	| **%f**     | \emph{et al.}        	|  -Henneken, Muench, Holm Nielsen, Blanco-Cuaresma, and Accomazzi<br>-Kurtz and Accomazzi                                            	|  -Henneken, Muench \emph{et al.}<br>-Kurtz and Accomazzi                                     	|
```

Comment:
Move this row up to the top, to be in alphabetical order with the rest of the examples

Reply:
done.